client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -18,6 +18,15 @@ export default class ReserveSubmitButton extends Vue {
   get entity_type_cd () {
     return newReqModule.entity_type_cd
   }
+  messages () {
+    if (Array.isArray(newReqModule.analysisJSON.issues)) {
```

Comment:
Just curious about this check here. Is this specifically to check if the data type of `newReqModule.analysisJSON.issues` is an Array?
Or is the aim to see if `.issues` is not empty? 

Curious if `Array.isArray()` would still return true if the value provided was an empty array.

Reply:
My intention was simply to verify it was an array.  `Array.isArray([])` does return true but I think this is the extent of the data checking needed here because I just wanted to be sure it was an Array of any sort before calling .map

Reply:
Fair enough!